app/views/includes/search-form.nunjucks

```diff
@@ -10,19 +10,13 @@
       {% if errorMessage %}
         <h2>{{ searchErrorLabel }}</h2>
         <p>{{ errorMessage }}</p>
-      {% elif gps.length === 0 %}
-         <h2>We can't find a surgery using '{{search}}'</h2>
-         <p>Check the text you entered is right. You get better results if you enter a full name. Or you can search with a postcode instead.</p>
       {% else %}
-        <h2>Find your GP surgery</h2>
-        <p>You need to do this to go to your booking system. Enter the name of your surgery, the name of your GP or a postcode.</p>
+        {{ noResultsMessage | safe }}
```

Comment:
An example to use would be searching <script>alert('Injected!');</script>. Google doesn't allow it to be posted and Safari just hangs. Searching by \<h1\>test\</h1\> _appears_ to sanitise the input.